.github/workflows/testingfarm.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: Testing farm tests
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  testingfarm:
+    name: "Run in testing farm"
+    runs-on: ubuntu-latest
+    if: |
+      github.event.issue.pull_request
+      && contains(github.event.comment.body, '[test]')
+      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v4
+      with:
+        ref: "refs/pull/${{ github.event.issue.number }}/head"
+        submodules: true
+
+    - name: Run the tests
+      uses: sclorg/testing-farm-as-github-action@v2
+      with:
+        compose: Fedora-39
+        api_key: ${{ secrets.TF_API_KEY }}
+        git_url: "https://gitlab.cee.redhat.com/rhinstaller/builders"
+        git_ref: "master"
+        path: "ansible"
```

Comment:
This will be supported by https://github.com/sclorg/testing-farm-as-github-action/pull/147

Reply:
This is already fixed and supported.